app/controllers/users_controller.rb

```diff
@@ -1,12 +1,14 @@
 class UsersController < ApplicationController
-  def show
-    @user = User.find_by id: params[:id]
-    return if @user
+  before_action :logged_in_user, only: [:index, :edit, :update, :destroy]
+  before_action :correct_user, only: [:edit, :update]
+  before_action :load_user, only: [:show, :edit, :update, :destroy]
```

Comment:
```suggestion
  before_action :load_user, only: %i(show edit update destroy)
  before_action :correct_user, only: %i(edit update)
```

ngoài ra
`before_action :load_user, only: [:show, :edit, :update, :destroy]  ` thành `before_action :load_user, except: %i(index new create)`